malaria.js

```diff
@@ -0,0 +1,62 @@
+var axios = require('axios');
+var reqUrl ='https://www.ebi.ac.uk/europepmc/webservices/rest/search';
+var format = require('./utils');
+
+exports.getWithYear = function (req, res){	
+	//End Date Func
+	var date = new Date();
+	var end = format.date;
+	const params = {
+		query: req.query.query,
+		date:`( FIRST_PDATE:[${req.query.start || req.start} TO ${req.query.end || end}])`,
+		sort: 'CITED desc',
+		pageSize:10,
+		synonym: true,
+		format: 'json' 
+	}
+	axios.get(reqUrl,{ params: params})
```

Comment:
@Firnaa5 Can be more simplified as `axios.get(reqUrl,{ params})`

Reply:
Simplified as axios.get(reqUrl,params)